LESS/mixins.less

```diff
@@ -1,8 +1,8 @@
 .side-rails() {
-    margin: 0 1rem;
+    margin: 0 5%;
```

Comment:
Percentages work here, nice switch.  You could also use and will see a lot of developers using `em` here.  `rem` here doesn't make sense since the `r` is a reference to the relationship to the relative font size and that is probably a relationship you don't want with regards to box-model metrics. Excellent switch, as you can never go wrong with some responsive percents.. just gives such a fluidity to your viewport transitions.